gmm_client/src/client.erl

```diff
@@ -1,94 +1,57 @@
 %%%-------------------------------------------------------------------
-%%% @author Svatopluk
+%%% @author Piotr Świderski
 %%% @copyright (C) 2021, <COMPANY>
 %%% @doc
 %%%
 %%% @end
 %%% Created : 10. maj 2021 16:54
 %%%-------------------------------------------------------------------
 -module(client).
--author("Svatopluk").
+-author("Piotr Świderski").
 
 
 %% API
--export([get_id_list/0, get_id_number/0, get_user_info/1, delete_user/1, add_user/1, test/0]).
+-export([get_id_list/0, 
+          get_user_info/1, 
+          delete_user/1, 
+          add_user/1]).
 
-test()->
-  ok.
+% CONST
+-define(URL, "localhost:8080/").
 
-url_address() ->
-  "localhost:8080/".
-
-% 
-
-get_request(URL)->
-  Method = get,
-  Headers = [],
-  Payload = <<>>,
-  Options = [],
-  {ok, StatusCode, RepsHeaders, ClientRef} = hackney:request(Method, URL, Headers, Payload, Options),
+request(Method, Url, Headers, Payload, Options )->
```

Comment:
W sumie to często wykorzystujesz tą funkcję z pustymi trzema ostatnimi argumentami, to można by zrobić to:
```erlang
request(Method, URL) -> request(Method, URL, [], <<>>, []).
```

Reply:
Jasne, i to jest bardzo fajne, tylko proponuję wprowadzenie uproszczonej wersji